app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -116,6 +116,8 @@ def get_load_balancer_pool_members
   end
 
   def get_load_balancer_listeners
+    # Initialize the data, otherwise the saving code will not catch this when empty
+    @data[:load_balancer_listeners] ||= []
```

Comment:
I wonder why I can merge this PR, as https://github.com/ManageIQ/manageiq-providers-amazon/pull/67 modifies the same file here?!